extensions/azurecore/src/account-provider/azureAccountProvider2.ts

```diff
@@ -305,7 +305,9 @@ export class AzureAccountProvider implements azdata.AccountProvider {
 
 		tenants = tenants.filter(t => t !== undefined);
 		if (tenants.length === 0) {
-			throw new Error(localize('azure.noTenants', "No azure tenants found. Failing..."));
+			const msg = localize('azure.noTenants', "No azure tenants found. Failing...");
```

Comment:
I'd suggest the text be something like `Failed to add account, there were no Azure tenants found.` Bit more clear that way.